front/src/gc-components/GCList.tsx

```diff
@@ -0,0 +1,16 @@
+import { DefaultMantineColor, List, MantineNumberSize } from "@mantine/core";
+import { ReactNode } from "react";
+
+interface GCListProps {
+    children: ReactNode;
+    size?: MantineNumberSize;
+    color?: DefaultMantineColor;
+}
+
+export const GCList = ({ size, color, children }: GCListProps) => {
```

Comment:
to the props list we also should add ...props, this gives to us possibilty to support all props from Mantine original component